test/cli_harness.sh

```diff
@@ -52,3 +52,23 @@ function fail {
 	echo; echo "FAILURE: $msg"
 	false
 }
+
+# compares two files and the inline source map
+function assert_identical_sourcemap {
+	actual="${1:?}"
+	expected_src="${2:?}"
+	expected_map="${3:?}"
+
+	# save the original file
+	mv "$actual" "$actual.orig"
+	# drop the inline source map
+	sed '\!sourceMappingURL!d' <"$actual.orig" >"$actual"
+	# decode and clean the inline source map
+	#  source maps contain system-dependent information (full paths)
+	#  so we drop all information we're not explicitly testing
+	node -r "$root/../node_modules/faucet-pipeline/lib/util/extract-sourcemap" -e "extractSourcemap('$actual.orig', ['version', 'names', 'mappings', 'sourcesContent'])" -p > "$actual.map"
```

Comment:
This line is veeeeeeeeery long 😆

Reply:
There are bunch of things we could extract variables for, like the path to `…/faucet-pipeline/lib` or `…/util`, or the list of attributes we test for.

Reply:
Good idea, yes.

Reply:
Indeed. Also, line continuations exist. 😃

Reply:
Yep. I'm also not sure if we should rather put the four attributes we are extracting into the source map extraction function instead of passing them as arguments... At least in the case of calling it as a script?